### ISSUE TITLE
update status in api.linkedtv.eu/mediaresource to CURATED

### ISSUE THREAD

**Issue by jthomsen** [linkedtv/editortool#6]
when changes from EditorTool have been saved (currently into own graph), the status of the mediaresource should be updated to CURATED

PUT /mediaresource/{mediaresourceid}
Body:

<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
    <mediaresource>
       <linkedTVStatus>
          <name>CURATED</name>
       </linkedTVStatus>
    </mediaresource>
for more details pls contact alisarioglu

**Comment by jthomsen** [comment]
in the light of https://github.com/linkedtv/editortool/issues/10 it might be appropriate to have even two status:
6 = CURATIONSTARTED
7 = CURATED (or CURATIONFINISHED)
in which case https://github.com/linkedtv/editortool/issues/13 could be changed to not longer allow the curation of videos which have been finished

**Comment by jblom** [comment]
don't really see why this status is useful, when it's already possible to filter out data with the "editor tool" provenance.

I think this state should be changed to PUBLISHED, meaning the content owner allows this programme to be published in the player. That would be very useful and is currently missing in the platform.

**Comment by jthomsen** [comment]
ok, think these are two different things; the status CURATED is just to easily keep track of which steps in the workflow have been processed; we can also rename it to PUBLISHED (in the end it's just a label assigned to a number) or in how far would be the status PUBLISHED different from CURATED (apart from the name)? 
However, this would suggest that there are no PUBLISHED videos without CURATION; do we really have this condition? Or else, in these cases we would have to set automatically the status right to PUBLISHED after ENRICHED

**Comment by rtroncy** [comment]
I would personally just keep the status CURATED if this went to the hands of an editor. The PUBLISHED status does not really make sense. Videos can be ENRICHED and not CURATED.
To summarize, I don't see why we need two status, one is enough. CURATED is a good label for it.

**Comment by jblom** [comment]
CURATED is just weird. When is a video curated? When there is one
annotation made by an editor? I'd say the editor wants to be able to say:
"this can be published" (can be viewed by the public), regardless of
whether it has been "curated". The player should only be able to load items
that were published by an editor.

If you want to keep the name CURATED, that's fine, but I'd use in the ET as
a "Publish" button.

2014-07-24 12:30 GMT+02:00 Raphael Troncy notifications@github.com:

> I would personally just keep the status CURATED if this went to the hands
> of an editor. The PUBLISHED status does not really make sense. Videos can
> be ENRICHED and not CURATED.
> To summarize, I don't see why we need two status, one is enough. CURATED
> is a good label for it.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/linkedtv/editortool/issues/6#issuecomment-49991830.

## 

_Jaap Blom_

Projectmedewerker R&D

_T_ 035 - 677 1930
_Aanwezig:_ ma,di,wo,do,vr
 http://www.beeldengeluid.nl _Nederlands Instituut voor Beeld en Geluid_
*
Media Parkboulevard 1, 1217 WE  Hilversum | Postbus 1060, 1200 BB  Hilversum | *
_beeldengeluid.nl_ http://www.beeldengeluid.nl

**Comment by jthomsen** [comment]
in a real production environment a further status PUBLISHED (but after CURATED, or ENRICHED, resp.) could IMHO indeed make sense, which would change the video and it's attached metadata to personalizable and publicly visible; but for us it's not necessary at the moment, as we don't distinguish these status

**Comment by jthomsen** [comment]
yes, actually your CURATED then means CURATION FINISHED which is when the Publish button is pressed; this would be then the right moment to set the status to CURATED; if we would like to add another status PUBLISHED then we would some rules saying:
non curated videos: ENRICHED -> PUBLISHED
curated videos: CURATED -> PUBLISHED
this would indeed make sense, because thus we could control whether a curation is expected or needed for these types of videos or not

**Comment by rtroncy** [comment]
To the question "when a video is curated", the simple answer is when the editor says so. There should not be a minimal number of edits to be in a curated state. A video which has been curated can be curated again, etc. I  don't see this as an end, so I don't understand why there should be the notion of start and finish. When a video enters the editor tool, even if it is for validating annotations/enrichments, it becomes CURATED, by definition.
